Tabloid/client/src/components/Tag/tagForm.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import { FormGroup, Input, Label, Button, Form } from "reactstrap";
+import { addTag } from "../../modules/tagManager";
+
+const TagForm = () => {
+    const history = useHistory();
+    const emptyTag = {
+        Name: "",
+    };
+
+    const [tag, setTag] = useState(emptyTag);
+
+    const handleInputChange = (evt) => {
+        const value = evt.target.value;
+        const key = evt.target.id;
+
+        const tagCopy = { ...tag };
+
+        tagCopy[key] = value;
+        setTag(tagCopy);
+    };
+    const handleSave = (evt) => {
+        evt.preventDefault();
+        addTag(tag).then((p) => {
```

Comment:
do you need a parameter (p) in this callback function?

Reply:
👍🏻